scripts/pre_commit_linter.py

```diff
@@ -538,11 +781,15 @@ def _lint_js_files(
         print 'There are no JavaScript files to lint.'
         return
 
-    print 'Total js files: ', num_js_files
+    if verbose_mode_enabled is True:
```

Comment:
You know, here and elsewhere -- if you're just printing one line, I really don't think you should hide that info behind a verbose flag, it's fine to just print it. There is a line further down where you print the number of python files if verbose mode is True and print "linting python files" if verbose mode is False. The former really doesn't take up much more space than the latter.

So, I would suggest putting things behind a verbose flag only if they span more than 1-2 lines. Otherwise just print them (even in non-verbose mode).

Reply:
Done